test/ts/logs/log_utils.ts

```diff
@@ -2,11 +2,17 @@ import * as ABIDecoder from "abi-decoder";
 import * as BigNumber from "bignumber.js";
 import * as _ from "lodash";
 
+// Extend ABIDecoder.DecodedeMethodParam to accept values of
+// type string and boolean
+interface DecodedMethodParam extends ABIDecoder.DecodedMethodParam {
+    value: any;
```

Comment:
Good instinct to create a new type -- we actually define the type definitions for `ABIDecoder` in this repo.  I would just change the type definition for `DecodedMethodParam` in `types/modules.d.ts` from `value: string;` to `value: string | boolean;`